hoot-core-test/src/test/cpp/hoot/core/io/ServiceHootApiDbReaderTest.cpp

```diff
@@ -101,53 +105,15 @@ class ServiceHootApiDbReaderTest : public CppUnit::TestFixture
     }
   }
 
-  //TODO: this is redundant with code in ServiceOsmApiDbReaderTest; move to test utils class
```

Comment:
While I was at it, I followed the suggestion of this comment and moved the code below to `ServicesDbTestUtils::createServiceTestMap` since it is exactly the same between this and `ServiceOsmApiReaderTest`.